modules/va_facilities/README.yml

```diff
@@ -19,9 +19,15 @@ info:
 
     ## Design
 
+    ### Base URLs
+    
+    The base URLs for the VA Facilities API in the various environments are:
+    - Development: `https://dev-api.va.gov/services/va_facilities/v0`
+    - Production: `https://api.va.gov/services/va_facilities/v0`
+
     ### Authorization
 
-    API requests are authorized through a symmetric API token, provided in an HTTP header with name "apikey".
+    API requests are authorized through a symmetric API token, provided in an HTTP header with name `apikey`.
```

Comment:
I'd love to know how to get that API key -- this is something that's I could pull from the config file? `credstash`? I followed along as someone without any background on this and its the only thing that I was a bit unclear about.

Clearly you just added formatting, but if you can clarify that would be helpful to me at least

Reply:
These docs are meant for users of the API, not developers of the API. An API key can be requested through this link: https://developer.va.gov/apply However, this only gets you access to dev. Access to the prod api is a different process. It would be nice to add the `apply` link here for clarity